<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamWidget</class>
 <widget class="QWidget" name="ParamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>597</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Viewer configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_38">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>show menu bar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonLoadSettings">
            <property name="text">
             <string>Load Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveSettings">
            <property name="text">
             <string>Save Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="switchSparkView">
         <property name="text">
          <string>Switch Spark View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollAreaViewer">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>357</width>
            <height>575</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widgetViewer" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_39">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="threeDGroupBox">
                <property name="title">
                 <string>3D Model</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_19">
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkBoxBB">
                   <property name="text">
                    <string>Bounding Boxes</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="checkBoxGhosts">
                   <property name="text">
                    <string>Ghosts</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBoxJoints">
                   <property name="text">
                    <string>Joints</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="checkBoxEnableLight">
                   <property name="text">
                    <string>Light Model</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="checkBoxDrawGoto">
                   <property name="text">
                    <string>Draw GoTo conf</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="sceneGroupBox">
                <property name="title">
                 <string>Scene</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_20">
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="checkBoxFloor">
                   <property name="text">
                    <string>Floor</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QCheckBox" name="checkBoxTiles">
                   <property name="text">
                    <string>Tiles</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QCheckBox" name="checkBoxShadows">
                   <property name="text">
                    <string>Shadow</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QCheckBox" name="checkBoxAxis">
                   <property name="text">
                    <string>Axis</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="checkBoxSmooth">
                   <property name="text">
                    <string>Smooth</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QCheckBox" name="checkBoxFilaire">
                   <property name="text">
                    <string>Filaire</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QCheckBox" name="checkBoxWalls">
                   <property name="text">
                    <string>Walls</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QCheckBox" name="checkBoxContour">
                   <property name="text">
                    <string>Contours</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="lightGroupBox">
                <property name="title">
                 <string>Light</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_25">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <item row="1" column="0">
                  <widget class="QWidget" name="widgetLightSliders_5" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_47">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxLightX"/>
                    </item>
                    <item row="0" column="2">
                     <widget class="QSlider" name="horizontalSliderLightX">
                      <property name="maximum">
                       <number>1000</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxLightY"/>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="label_X">
                      <property name="text">
                       <string>X</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="label_Y">
                      <property name="text">
                       <string>Y</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QSlider" name="horizontalSliderLightY">
                      <property name="maximum">
                       <number>1000</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxLightZ"/>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="label_Z">
                      <property name="text">
                       <string>Z</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QSlider" name="horizontalSliderLightZ">
                      <property name="maximum">
                       <number>1000</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkBoxDrawLightSource">
                   <property name="text">
                    <string>Draw Light Source</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widgetLightButtons" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_40">
                 <item>
                  <widget class="QPushButton" name="pushButtonRestoreLight">
                   <property name="text">
                    <string>Restore Light</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonSaveVideo">
                   <property name="text">
                    <string>save video</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonRestoreView">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Restore View</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>128</width>
        <height>168</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Spark Environment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_41">
       <item>
        <widget class="QLineEdit" name="sparkStatusLineEdit">
         <property name="text">
          <string>not updating</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="sparkCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>refresh</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sparkPosterlabel">
         <property name="text">
          <string>Poster name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sparkPosterLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>sparkEnvironment</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sparkSaveSceBut">
         <property name="text">
          <string>Save scenario</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sparkVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>45</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Niut</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_42">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
